epubcheck-adapter/pom.xml

```diff
@@ -34,13 +41,11 @@
       <groupId>org.daisy.pipeline</groupId>
       <artifactId>calabash-adapter</artifactId>
     </dependency>
-
     <!-- EpubCheck -->
     <dependency>
-  	  <groupId>org.idpf</groupId>
-  	  <artifactId>epubcheck</artifactId>
-      <version>4.0.1</version>
-  	</dependency>
+      <groupId>org.daisy</groupId>
+      <artifactId>epubcheck</artifactId>
```

Comment:
I'm not entirely sure I understand why the version is moved to the BOM here. This dependency is very specific to this particular module, so it might be easier to maintain if kept here?